clearwater-cluster-manager.root/etc/cron.hourly/clearwater-cluster-manager-log-cleanup

```diff
@@ -3,4 +3,5 @@ ONE_HUNDRED_MEG=$((100 * 1024 * 1024))
 max_log_directory_size=$ONE_HUNDRED_MEG
 . /etc/clearwater/config
 
-python2.7 /usr/share/clearwater/bin/log_cleanup.py /var/log/clearwater-cluster-manager --prefix cluster_manager --maxsize $max_log_directory_size
+python2.7 /usr/share/clearwater/bin/log_cleanup.py /var/log/clearwater-cluster-manager --prefix cluster-manager --maxsize $max_log_directory_size
+python2.7 /usr/share/clearwater/bin/log_cleanup.py /var/log/clearwater-etcd --prefix mark_node_failed --maxsize $max_log_directory_size
```

Comment:
Is this path write for the mark_node_failed logs? (And also, is the prefix definitely right?) Should this go in the clearwater-etcd-log-cleanup script? Or alternatively if this is a cluster manager script, should it log to the cluster-manager directory?

Either way, I think this script should only be cleaning up one directory